### PR TITLE
Documentation/actualize windows compilation

### PR DESCRIPTION
**Pull request by KVS85**
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-documentation/226/

@Simran-B @dothebart 
I absolutely not sure about the last part related to documentation prerequisites. Possibly, it could be removed.

### CODE REVIEW

**Comment by KVS85** [comment]
http://web01.arangodb.biz/documentation/actualize-windows-compilation/Cookbook/Compiling/Windows.html

**Comment by Simran-B** [comment]
@KVS85 the markdown-pp part can be removed:

```
Markdown-pp:

git clone https://github.com/triAGENS/markdown-pp.git
cd markdown-pp
python setup.py install
```

Regarding ditaa: You can build the docs without it, but two graphics will be missing. The problem with ditaa is that you need to write a wrapper script to run the command as `ditaa ...` (ditaa itself is only a jar file which needs to be run with the `java` command). We don't explain how to set this up, so we could as well remove ditaa from the list. Can't recommend to build docs under Windows anymore anyway.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-documentation/253/

http://web01.arangodb.biz/documentation/actualize-windows-compilation/Cookbook/Compiling/Windows.html

File: Documentation/Books/Cookbook/Compiling/Windows.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ Compiling ArangoDB under Windows
...
 
 Solution
 --------
 
-With ArangoDB 3.0 a complete cmake environment was introduced. This also streamlines the dependencies on windows.
+With ArangoDB 3.0 a complete cmake environment was introduced. This also streamlines the dependencies on Windows.
 We suggest to use [chocolatey.org](https://chocolatey.org/) to install most of the dependencies. For sure
 most projects offer their own setup & install packages, chocolatey offers a simplified way to install them
 with less user interactions. You can even use chocolatey via the brand new
 [ansibles 2.0 winrm facility](http://docs.ansible.com/ansible/intro_windows.html)
```
Comment (sleto-it) [inline Documentation/Books/Cookbook/Compiling/Windows.md]: current version seems 2.7 - link is old, the new one perhaps is https://docs.ansible.com/ansible/latest/user_guide/windows.html . I suggest we remove "brand new"

File: Documentation/Books/Cookbook/Compiling/Windows.md [hunk 2]
```diff
@@ -4,20 +4,20 @@ Compiling ArangoDB under Windows
 Problem
 -------
 
-I want to compile ArangoDB 3.0 and onwards under Windows.
+I want to compile ArangoDB 3.4 and onwards under Windows.
 
-**Note:** For this recipe you need at least ArangoDB 3.0;
-For ArangoDB version before 3.0 look at the [old Compiling ArangoDB under Windows](https://docs.arangodb.com/2.8/Cookbook/CompilingUnderWindows.html).
+**Note:** For this recipe you need at least ArangoDB 3.4;
+For ArangoDB version before 3.4 look at the [old Compiling ArangoDB under Windows](https://docs.arangodb.com/3.0/Cookbook/CompilingUnderWindows.html).
```
Comment (sleto-it) [inline Documentation/Books/Cookbook/Compiling/Windows.md]: the link does not exist

File: Documentation/Books/Cookbook/Compiling/Windows.md [hunk 3]
```diff
@@ -4,20 +4,20 @@ Compiling ArangoDB under Windows
...
 --------
 
-With ArangoDB 3.0 a complete cmake environment was introduced. This also streamlines the dependencies on windows.
+With ArangoDB 3.0 a complete cmake environment was introduced. This also streamlines the dependencies on Windows.
 We suggest to use [chocolatey.org](https://chocolatey.org/) to install most of the dependencies. For sure
 most projects offer their own setup & install packages, chocolatey offers a simplified way to install them
 with less user interactions. You can even use chocolatey via the brand new
 [ansibles 2.0 winrm facility](http://docs.ansible.com/ansible/intro_windows.html)
-to do unattended installations of some software on windows - the cool thing linux guys always told you about.
+to do unattended installations of some software on Windows - the cool thing Linux guys always told you about.
```
Comment (sleto-it) [inline Documentation/Books/Cookbook/Compiling/Windows.md]: "the cool thing Linux guys always told you about." - probably we should remove this comment

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-documentation/258/

**Pull request merged**